cli/loader_lib.c

```diff
@@ -79,24 +79,28 @@ static void * lookup_symbol(const void * lib_handle, const char * symbol_name) {
 }
 
 char lib_dir[PATH_MAX];
-const char * get_libdir()
+JL_DLLEXPORT const char * jl_get_libdir()
 {
+    // Reuse the path if this is not the first call.
+    if (lib_dir[0] != 0) {
+        return lib_dir;
+    }
 #if defined(_OS_WINDOWS_)
     // On Windows, we use GetModuleFileNameW
     wchar_t libjulia_path[PATH_MAX];
-    HMODULE libjulia_internal = NULL;
+    HMODULE libjulia = NULL;
 
     // Get a handle to libjulia internal
     if (!utf8_to_wchar(LIBJULIA_NAME, libjulia_path, PATH_MAX)) {
         jl_loader_print_stderr3("ERROR: Unable to convert path ", LIBJULIA_NAME, " to wide string!\n");
         exit(1);
     }
-    libjulia_internal = LoadLibraryW(libjulia_path);
-    if (libjulia_internal == NULL) {
+    libjulia = LoadLibraryW(libjulia_path);
+    if (libjulia == NULL) {
         jl_loader_print_stderr3("ERROR: Unable to load ", LIBJULIA_NAME, "!\n");
         exit(1);
     }
-    if (!GetModuleFileName(libjulia_internal, libjulia_path, PATH_MAX)) {
+    if (!GetModuleFileName(libjulia, libjulia_path, PATH_MAX)) {
```

Comment:
Should this be `GetModuleFileNameW` as the comment at the start of the Windows block says?

Reply:
I believe since we pass `-municode` it should automatically get `#define`'ed to that, but it doesn't hurt so go ahead and change it if you like.